src/app/search/FilterHelp.tsx

```diff
@@ -16,8 +27,15 @@ function keywordsString(keywords: string | string[]) {
   return keywords;
 }
 
-export default function FilterHelp() {
-  const searchConfig = useSelector(searchConfigSelector).filtersMap;
+export default function FilterHelp({ loadouts }: { loadouts?: boolean }) {
```

Comment:
nit: since this is likely going to be opened up to mods, have an explicit type or context field

```suggestion
export default function FilterHelp({ type }: { type: 'items' | 'loadouts' }) {
```

Another options could be to pass in the context selector (I think this is what I may have tried in the past, but I don't remember if I liked it)